tests/test_mwss_transform.py

```diff
@@ -579,3 +581,25 @@ def test_tempdir(self):
             tfr.pack(img_seq=itertools.count())
         except KeyError:
             self.fail("TODO: define pages of survey.")
+
+    def test_image_sequence_number(self):
```

Comment:
Can we add another unit test that covers the no `seq_nr` scenario to make sure the error deosn't make its way downstream?